dfm/data/load.py

```diff
@@ -16,7 +32,7 @@ def load_tweets(dedupe=False):
     data_path = os.path.join(f_path, "..", "data")
     sys.path.append(data_path)
     from dedupe import min_hash_deduper, duplicate_filter
-    from utils import to_datetime
+    from dfm.data.utils import to_datetime
```

Comment:
Shouldn't we stick all imports to the top of the scripts, to make it easier to get an overview of the code?

Reply:
I agree. Do not know why it was included in the function, nor why one would do that. Have moved it to the top.